### DISCUSSION TITLE
View messages via web page

### DISCUSSION THREAD

**Discussion by mhaase79** [nextgenhealthcare/connect#4693] [Q&A]
I may be creating a duplicate discussion, but I tried searching but couldn't find what I was after - I apologize for this. What I'm trying to do is create a web page where the messages of a channel can be viewed and or searched without requiring the user to sign in. If anyone can post examples of where to start that would be greatly appreciated.

Thank you,
Matt

**Comment by pacmano1** [comment]
That probably means calling Mirth's REST API and building your own web user interface using those responses.  Not trivial.

**Comment by rjkroll** [comment]
You can try looking at, using, Mirth's API via :  https://localhost:8443/api/         (or the servername if you are remote)

**Comment by tonygermano** [comment]
If the data in the messages is PHI or other protected information, you may want to reconsider not requiring a login.
